README.md

```diff
@@ -16,6 +16,23 @@ kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE exec -it $POD_NAME bash
 kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE delete deployment debugging-tools
 ```
 
+## SSH Server for reverse tunneling
+
+This repo also contains the Dockerfile for a ssh server, useful for making reverse tunnels to achieve a kind of "kubectl reverse-port-forward" behavior, and be able to expose services running in your local machine to the rest of the cluster. This server requires that you mount a `id_rsa.pub` key into `/root/.ssh/authorized_keys` to be able to log in.
```

Comment:
FYI I saw this a while ago, and I thought it was pretty awesome

`curl https://github.com/your-username.keys >> curl https://github.com/your-username.keys >> ~/.ssh/authorized_keys`